docs/SUMMARY.md

```diff
@@ -4,15 +4,21 @@
 
 ## Guides
 
-* [Aragon Basics](guides/aragon-basics.md)
 * [Getting started](guides/getting-started.md)
+* [Aragon Basics](guides/aragon-basics.md)
 
 ## Advanced
 
 * [Custom subgraph queries](advanced/low-level-queries.md)
```

Comment:
```suggestion
* [Custom Subgraph queries](advanced/low-level-queries.md)
```

TheGraph capitalizes these :)

Reply:
oh ok, I have a search & replace to make then!

Reply:
Done, with a few other things: https://github.com/aragon/connect/pull/103/commits/0f41155420460c69b74a1e41187a00a6b038c273